backend/app/rest/api/rest_test.go

```diff
@@ -277,7 +277,7 @@ func TestRest_parseError(t *testing.T) {
 }
 
 func startupT(t *testing.T) (ts *httptest.Server, srv *Rest, teardown func()) {
-	log.Setup(log.CallerFile, log.CallerFunc, log.Msec, log.LevelBraces)
+	log.Setup(log.CallerFile, log.CallerFunc, log.Msec, log.LevelBraces, log.Debug)
```

Comment:
such a change affects all the tests and makes them all verbose. I'd suggest doing it in your test directly and not modifying output of everything else

Reply:
initially, I had it with log.Debug but it made such enormous build logs I turned it off